src/ConvertInputFormat/main.cpp

```diff
@@ -253,15 +255,20 @@ bool processErrors(std::unique_ptr<IdfParser> const &idf_parser, std::unique_ptr
         displayMessage(warning);
     }
     for (auto const &error : validation_errors) {
+        if (isDDY) {
+            if ((error.find("Missing required property 'Building'") != std::string::npos) ||
+                (error.find("Missing required property 'GlobalGeometryRules'") != std::string::npos)) {
+                continue;
+            }
+        }
+        hasValidationErrors = true;
```

Comment:
And here's the filtering for the DDY case: don't require Building or GlobalGeometryRules

Reply:
Interesting.  In a more ideal world, I guess we'd just not require those properties when validating the DDY.  But we don't really have a way to do that, at least not quickly.  So instead, we let those errors occur, but then filter them out here.  I anticipate this will need to change at most once every I dunno, 5 years, so this is fine.